packages/flutter_test/lib/src/binding.dart

```diff
@@ -1784,15 +1784,6 @@ class _LiveTestRenderView extends RenderView {
     onNeedPaint();
   }
 
-  @override
-  bool hitTest(HitTestResult result, { required Offset position }) {
-    final Matrix4 transform = configuration.toHitTestMatrix();
```

Comment:
Removing this seems to break the logic where you can just `flutter run` a test, tap on the screen and it would tell you potential finders for widgets that are under your finger. (Looks like we don't have a test for that behavior, though 😞.) Maybe the transform logic has to move to a place where it only triggers for real pointer events and not for test-simulated ones?

/cc @dkwingsmt since the PR author bisected that this broke with https://github.com/flutter/flutter/pull/64846, which changed how test-simulated and real pointer events are routed in tests.

Reply:
I'm adding a unit test for the behavior you described and let's see if we can fix this issue without breaking it.

Reply:
I believe the tests where added in https://github.com/flutter/flutter/pull/83337. @prefanatic Can you rebase this PR to the latest master and see if this is breaking anything?

Reply:
@goderbauer Thanks, I've rebased the PR and pushed, looks green.